src/gui/translationshelper.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) by Duncan Mac-Vicar P. <duncan@kde.org>
+ * Copyright (C) by Klaas Freitag <freitag@owncloud.com>
+ * Copyright (C) by Daniel Molkentin <danimo@owncloud.com>
+ * Copyright (C) by Fabian Müller <fmueller@owncloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+#include "translationshelper.h"
+
+#include "config.h"
+#include "theme.h"
+
+#include <QApplication>
+#include <QLoggingCategory>
+#include <QDir>
+#include <QDirIterator>
+
+namespace OCC {
+
+Q_LOGGING_CATEGORY(lcTranslationsHelper, "gui.translationshelper", QtInfoMsg)
+
+const QString TranslationsHelper::TRANSLATIONS_FILE_PREFIX = "client_";
```

Comment:
Use an anonymous namespace for constants.
```
namespace {
const QString TranslationFilePrefixC() { return QStringLiteral("client_");}
}
```
For legacy reasons our const values end with a bit C even so they are no longer c strings.
Try to avoid global variables https://github.com/KDE/clazy/blob/master/docs/checks/README-non-pod-global-static.md .

Reply:
The prefix value is actually needed in another location as well (`application.cpp`, `setupTranslations`). I forgot to integrate it there so far. Do you want me to create the anonymous namespace in the header then? So far, the namespace method is only used in implementation but not header files.